### PR TITLE
Rspec

### PR DESCRIPTION
**Pull request by angelacuahutle**
For this project, we created various test cases for the Enumerable Methods and fix the syntax errors

### CODE REVIEW

File: spec/enumerables_spec.rb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  #   it 'filters an array of objects' do
+  #     expect([1, 2, 3].my_select { |n| n.&:even? }).to eql(2)
+  #   end
+  # end
+
+  # describe my_all? do
+  #   it 'returns true if objects satisfy the given condition, else it returns false' do
+  #     expect([1, 2, 3].my_all? { |n| n < 4 }).to eql(true)
+  #   end
+  # end
```
Comment (enaburekhan) [inline spec/enumerables_spec.rb]: Remove commented-out codes from the codebase. They make the codebase to be untidy, difficult to read and understand.

**Review by enaburekhan — CHANGES_REQUESTED** [review]
# Status: Changes required:full_moon:
**Hi team!
Great job on this project:clap::clap:.**

**major highlights**

- Specs for each method are wrapped in one describe block.
- There's one `it` block for each expected behavior.

**To Improve**

- [ ] Make `development` the default branch. "angelacuahutle wants to merge 15 commits into development from rspec" and not into enumerables.

- [x] One of the tests is failing on rspec. Kindly fix it.

- [x] There's currently no test for `multiply_els` method. Kindly write tests for it as well.

- [x] Currently, all the tests cases cover positive cases. Also account for negative test case 
scenarios.  A negative test case can always start like so: " it does not create the object if required attributes are missing... "

- [x] Update the readme to show how to run the test. For example, the test is run using `rspec` and not enumerables.rb. The test directory has -rspec extension. But the readme still shows the old project directory.

- [x] Remove commented-out code from the codebase as already highlighted below in the PR. You can use git history to view old commits using `git log .` instead of leaving them on the codebase. 

- [x] Do not make any changes in .rubocop.yml config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.

[Optional ]
- Move the Gemfile and Gemfile.lock to the root directory. 

Make the above-required changes and when you are ready, request another review.
For questions/comments, kindly drop them :point_down: and ping me on slack @ Eric Enaburekhan to respond.
Happy coding:blush:
Eric Enaburekhan

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @charlyeneh and @craigs40 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed.

##

Kindly work on the suggested changes 👇.
- [x] Remove `p` statements from `enumerables.rb`.
- [x] Do not make any changes in .rubocop.yml config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by Dipeshtwis — APPROVED** [review]
Hi **Shay Craig & Angela Cuahutle Navarro**, I'm Dipesh your reviewer for this PR 👩‍🚀 

Good job by removing the `p` statement from `enumerables.rb` and using right `.rubocop.yml` file.

-------

## **Your project is approved! 🎊 🥅 💛**

**You may close this Pull Request and merge the changes**


---------

    
    
**Keep being awesome devs and good luck with your next project!**

---------

Cheers and Happy coding!👏👏👏

Code review by **Dipesh Kumar**

(Feel free to contact me on slack by the handle of  **deep** if you have any additional question).
Happy Coding
[Dipesh Kumar](https://deeptwist.tk)
[Github](https://github.com/Dipeshtwis)

**Pull request closed without merging**